### DISCUSSION TITLE
How do you run this?

### DISCUSSION THREAD

**Discussion by StallmanSlave** [lucidrains/musiclm-pytorch#4] [Q&A]
readme doesn't have install instructions. I'm assuming there's prerequisites such as pytorch. I'm a Layman to this stuff.
Is there a simple local html based UI or is the project too new for that and requires manual text input via a python/terminal.

**Comment by evanshlom** [comment]
No one's going to make it that easy because there's not enough money in it to risk getting sued by the entire music industry

**Reply by jojicelf** [reply]
ayo grimes just sayed its cool with her
